core/templates/domain/exploration/SolutionObjectFactory.ts

```diff
@@ -149,7 +144,6 @@ export class SolutionObjectFactory {
   // TODO(ankita240796): Remove the bracket notation once Angular2 gets in.
   /* eslint-disable dot-notation */
   createNew(
-  /* eslint-enable dot-notation */
```

Comment:
Why remove this?

Reply:
there was no use of this.

Reply:
Okay, I was just confused since previously the stuf on line 150 was kept.